src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -707,7 +700,7 @@ void NetworkOPsImp::processHeartbeatTimer ()
 
```

Comment:
I agree this is confusing but would prefer modifying as a separate change, as I'm less sure the consequences.  

On first, pass, I'm not seeing how flapping would happen; I would think both calls for `setMode(omCONNECTED)` would generally end up setting the same mode, unless we are close to the 1 minute ledger age boundary.